code/index.html

```diff
@@ -24,17 +26,28 @@ <h1>Does the person have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
+          <option selected disabled default>Start guessing!</option>
```

Comment:
The “selected disabled default” option in the code was a smart thing to put here. 🤓

One suggestion though is to have it stand by itself in the drop down menu, e.g. put this on top of optgroup hair color instead of having it inside of the optgroup hair color.